index.html

```diff
@@ -20,23 +20,29 @@
     </head>
     <body>
        <div class="container">
-           <div class="container-top">
-               <div class="owner">John Doe</div>
-               <div class="menu-button"><i class="fa fa-bars"></i></div>
-            </div>
-            <div class="name">Hey There. I'm Leonard</div>
-            <div class="my-title">I am a Software Developer</div>
-            <div class="info">I can help you build a product,feature or website.Look through some of my work and experience! if you like what you see and have a project you need coded dont hesitate to contact me</div>
-            <div class="socials">
-                <i class="fa fa-github fa-2x" aria-hidden="true"></i>
-                <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
-                <i class="fa fa-linkedin fa-2x" aria-hidden="true"></i>
-                <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+           <div class="header">
```

Comment:
- [ ] Good job so far 👍 Instead of using a generic `div` tag here, a `header` tag would be better. So you would have something like this: `<header class="header">...</header>`